pkgs/applications/printing/braille-printer-app/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoreconfHook
+, pkg-config
+, cups
+, liblouis
+}:
+
+stdenv.mkDerivation rec {
+  pname = "braille-printer-app";
+  version = "2.0b1";
+
+  src = fetchFromGitHub {
+    owner = "OpenPrinting";
+    repo = "${pname}";
+    rev = "9304787c0797c5d99a5a0d17493c9c2b21865bc8";
+    hash = "sha256-kq1zIqp26I5/4De5PxvDEulhlzn5mQ2HnDNzO6H4Dac=";
+  };
+
+  nativeBuildInputs = [ autoreconfHook pkg-config ];
+  buildInputs = [
+    cups
+    liblouis
+  ];
+
+  postConfigure =
+    ''
+      # Patch shebangs of generated build scripts
+      patchShebangs filter
+    '';
```

Comment:
```suggestion
  postConfigure = ''
    # Patch shebangs of generated build scripts
    patchShebangs filter
  '';
```

Reply:
resolved in fixup commits (to be rebased, pending successful ofborg build)